tests/test_regrid.py

```diff
@@ -438,6 +438,26 @@ def test_regrid(self):
         assert "lon_bnds" in output
         assert "time_bnds" in output
 
+    @pytest.mark.parametrize(
+        "name,value,attr_name",
+        [
+            ("periodic", True, "_periodic"),
+            ("extrap_method", "inverse_dist", "_extrap_method"),
+            ("extrap_method", "nearest_s2d", "_extrap_method"),
+            ("extrap_dist_exponent", 0.1, "_extrap_dist_exponent"),
+            ("extrap_num_src_pnts", 10, "_extrap_num_src_pnts"),
+            ("ignore_degenerate", False, "_ignore_degenerate"),
+        ],
+    )
```

Comment:
I didn't know this pytest decorator exists. This is neat for easily testing different function/method parameters!